client/packages.go

```diff
@@ -112,6 +114,42 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+func setPrice(snap *Snap, currency string) {
```

Comment:
According to @pete-woods, on the phone the price displayed only ever makes use of the suggested currency returned from the store. Knowing prices in other currencies is then redundant.

There's the theoretical ability to specify a preferred currency on your device but I think it's much simpler when your preference is associated with your store account and then the likes of snapweb and `snap find` just parrot the price given to them from the client lib.